locales/fluent/base/en.ftl

```diff
@@ -12,4 +48,57 @@ logout = Logout
 register = Register
   .toast = You have been registered.
 
-create-account = Create an account
+specifier = Email or Username
+  .placeholder = Enter email or username...
+
+username = Username
+  .placeholder = Enter username...
+  .info = You will be able to change this later.
```

Comment:
Unrelated to this PR, but I wonder if we should show what the user slug will be during registration. Especially since if the slug conflicts the user can't be created.

Reply:
Good idea, actually. I'll do that.